src/test/java/com/jQueryTest/JQueryFirstAssignmentTest.java

```diff
@@ -1,7 +1,533 @@
 package com.jQueryTest;
 
-/**
- * Created by gturd on 5/18/2021.
- */
+
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
 public class JQueryFirstAssignmentTest {
-}
+    WebDriver driver;
+    SoftAssert softAssert = new SoftAssert();
+
+    @BeforeClass
+    public void login() {
+        //set up webdriver
+        System.setProperty("webdriver.chrome.driver",
+                "C:\\Users\\gturd\\IdeaProjects\\FirstAssignment\\webdriver\\chromedriver.exe");
```

Comment:
create a "drivers" package in your project and put your chromedriver into the package. 
then use this next time when u initializing your driver :  
 System.setProperty("webdriver.chrome.driver", System.getProperty("user.dir") + "//drivers//chromedriver");
(please memorize this line)

Reply:
because you are using windows, it should be like this:
System.setProperty("webdriver.chrome.driver", System.getProperty("user.dir") + "//drivers//chromedriver.exe");
sorry for the confusion.